packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -81,12 +81,21 @@ class SnackBarAction extends StatefulWidget {
   /// The [label] and [onPressed] arguments must be non-null.
   const SnackBarAction({
     Key key,
+    this.textColor,
+    this.disabledTextColor,
     @required this.label,
     @required this.onPressed,
   }) : assert(label != null),
        assert(onPressed != null),
        super(key: key);
 
+  /// The button label color. If not provided, defaults to accentColor.
+  final Color textColor;
+
+  /// The button disabled label color. This color is shown after the
+  /// snackBarAction is dismissed.
```

Comment:
similarly here

Reply:
What is here? Don't think there's any named variable?